libs/apnos/apnos.py

```diff
@@ -0,0 +1,140 @@
+import paramiko
+
+
+class APNOS:
+
+    def __init__(self, credentials=None):
+        self.owrt_args = "--prompt root@OpenAp -s serial --log stdout --user root --passwd openwifi"
+        if credentials is None:
+            exit()
```

Comment:
Think we should let the user know what's wrong and exit with a non zero error code.

Reply:
this is done

Reply:
prolly we will do in other locations as well

Reply:
How is this resolved?

Reply:
Yes i did that

Reply:
I added a print statement